test/components/attributes/ModificationButtonTest.js

```diff
@@ -0,0 +1,60 @@
+/*eslint-env node, mocha */
+/*global expect */
+/*eslint no-console: 0*/
+'use strict';
```

Comment:
We don't actually need these. When I first generated the app billions of years ago (last year); the generator added a lot of this pointless boilerplate garbage.

I'll remove this stuff later. But just know we won't need it going forward.